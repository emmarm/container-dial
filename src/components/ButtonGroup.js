import React from 'react';
import PropTypes from 'prop-types';
import styled from 'react-emotion';

const Group = styled('div')({
  display: 'flex',
  justifyContent: 'space-around',
  margin: '25px'
});

const ButtonGroup = ({ children }) => <Group>{children}</Group>;

ButtonGroup.propTypes = {
  children: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
};

export default ButtonGroup;
