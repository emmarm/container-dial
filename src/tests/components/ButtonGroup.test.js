import React from 'react';
import { shallow } from 'enzyme';

import ButtonGroup from '../../components/ButtonGroup';

describe('ButtonGroup component', () => {
  const wrapper = shallow(<ButtonGroup />);

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('renders a Group div', () => {
    const group = wrapper.find('Styled(div)');
    expect(group.length).toBe(1);
  });
});
