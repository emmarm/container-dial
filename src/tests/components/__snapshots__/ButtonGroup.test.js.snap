// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup component matches snapshot 1`] = `<Styled(div) />`;
